src/components/ComboButton/ComboButton.js

```diff
@@ -124,34 +124,36 @@ const ComboButton = ({ children, className, direction }) => {
       ref={wrapper}
       data-floating-menu-container
     >
-      {primaryActionWithProps}
-
-      {overflowMenuItemWithProps !== undefined && (
-        <OverflowMenu
-          className={classnames(
-            // Button-specific classes for styling:
-            buttonNamespace,
-            `${prefix}--btn`,
-            `${prefix}--btn--primary`,
-
-            // Button as a child of combo button:
-            `${namespace}__button`,
-
-            // Overflow menu as a child of combo button:
-            `${namespace}__overflow-menu`
-          )}
-          direction={direction}
-          menuOffset={getMenuOffset}
-          menuOptionsClass={`${prefix}--list-box__menu`}
-          onClose={() => setIsOpen(false)}
-          onOpen={() => setIsOpen(true)}
-          renderIcon={() =>
-            isOpen ? overflowIcon(ChevronUp16) : overflowIcon(ChevronDown16)
-          }
-        >
-          {overflowMenuItemWithProps}
-        </OverflowMenu>
-      )}
+      <div className={`this ${namespace}__group`}>
```

Comment:
- [ ] Would you mind removing the `this` class if it's not doing anything?

Reply:
@SimonFinney I disagree on this... I feel like before, we've talked about attaching classes to wrappers, even if *we* arent' necessarily styling said wrappers, in order to allow end users to more easily target those wrappers, if needed. I believe this is Carbon's approach as well.

Reply:
Oops. Typo. Sorry!